tools/klockstat.py

```diff
@@ -431,7 +431,7 @@ def display(sort, maxs, totals, counts):
 
         avg = totals[k].value / counts[k].value
 
-        print("%40s %10lu %6lu %10lu %10lu" % (caller, avg, counts[k].value, maxs[k].value, totals[k].value))
+        print("%40s %10lu %6lu %10lu %10lu" % (caller.decode('utf-8'), avg, counts[k].value, maxs[k].value, totals[k].value))
```

Comment:
Could you provide a second argument to prevent a UnicodeDecodeError, similar to how other tools handle it?
`
caller.decode('utf-8', 'replace')
`

Reply:
Sure.

Reply:
Hi @ekyooo thanks for the review, I have updated the PR. Do let me know if anything else is required.

Reply:
The change of this patch LGTM. 
But additionally, I think it's good to switch %lu to %d as Python doesn't support it as I know.
Thank you.

Reply:
Done. Thanks you